pkgs/applications/misc/librecad/default.nix

```diff
@@ -18,8 +18,11 @@ stdenv.mkDerivation rec {
   qmakeFlags = [ "MUPARSER_DIR=${muparser}" "BOOST_DIR=${boost.dev}" ];
 
   installPhase = ''
-    mkdir -p $out/bin $out/share
-    cp -R unix/librecad $out/bin
```

Comment:
was there a single binary ? the new version just keeps bin/librecad.

Reply:
Yes there was, not sure why they used `-R` before.

Reply:
Correct, the only file in `bin` is `librecad`.